public/js/user-actions.js

```diff
@@ -13,13 +16,56 @@ function UserActions(localDispatcher) {
 }
 
 
-UserActions.prototype.setCurrentUser = function(user) {
-  console.log('UserActions: setting current user', user);
-  this.dispatcher.dispatch({
-    actionType: 'set-user',
-    user: user,
-  });
-};
+UserActions.prototype = assign({}, {
```

Comment:
Correct, it isn't doing anything here. A previous version was inheriting from another object which is why I was using it.